editors/code/src/commands.ts

```diff
@@ -477,7 +480,8 @@ export function applyActionGroup(_ctx: Ctx): Cmd {
         vscode.commands.executeCommand(
             'rust-analyzer.resolveCodeAction',
             selectedAction.arguments,
-        );
+        )
+            .then(() => { }, console.error);
```

Comment:
Here and elsewhere, seems like we can `await`? Actually fixing the problem is preferable to sweeping it under the rug ;)

Reply:
@matklad doesn't this require us to change all of the parent function signatures to async?

Reply:
parent function here is already async, no?

Reply:
@matklad I made some few changes, can u check again? also I noticed there is a type called `Cmd` which is a normal function but we are using it as if we are returning an async function (example: `commands.ts/analyzerStatus()`) and at the end we are calling them like normal functions so we don't `await` or `catch() `them. doesn't this cause the same trouble?